lib/module.js

```diff
@@ -74,6 +74,15 @@ function Module(id, parent) {
 }
 module.exports = Module;
 
+const builtinModules = Object.keys(process.binding('natives'))
+  .filter((module) => {
+    return !module.startsWith('internal/') &&
+      !module.startsWith('v8/') &&
+      !module.startsWith('node-inspect/');
+  });
```

Comment:
Is all the extra filtering beyond `startsWith("internal/")` needed? I think other places in Node just check for "internal/" so while things like `v8/` and `node-inspect/` aren't super great they're built-in and accessible via `require()` which is what really matters.

Reply:
Not anymore: 0e10717e437eeb05e57d86e823777f418f00c478

Reply:
So it will now throw a deprecation warning but they're still accessible?
I think the bar should be:
Can the module be required?
Yes - then it's in the list
No - then it's not

A deprecation warning but still being able to be required means it would be on the list because folks can turn off deprecation warnings.